demo/thumbnail/basic/context.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
It's autogenerated by sui-studio.

In case you already knew it: I don't want to check if every file that sui-studio generates makes sense or not, I assume that they do or at least that they would tell me that they are optional (like the props in the src/index.js file), therefore this file still exists.